commands/solenoid/SolenoidSet.java

```diff
@@ -1,18 +1,21 @@
 package org.usfirst.frc4904.standard.commands.solenoid;
 
 import java.util.function.BooleanSupplier;
+import java.util.Set;
 
 import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
 import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
-import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
 
 /**
  * Command to set the state of a SolenoidSubsystem
  */
-public class SolenoidSet extends Command {
+public class SolenoidSet implements Command {
```

Comment:
I think this needs to extend `CustomCommandBase`.

Reply:
see #245.